specs/darwin/keychain_acl_apps.table

```diff
@@ -0,0 +1,10 @@
+table_name("keychain_acls")
+description("Applications that have ACL entries in the keychain")
+schema([
```

Comment:
I'd suggest: label, description, authorizations, app_path, path. Renaming `keychain_path` to path matches `keychain_items`. Have a non-descript "path" column names is more or less the MO for tables. Where multiple paths are used the secondary path is usually more-well-named.